atlasdb-client/src/main/java/com/palantir/atlasdb/schema/stream/StreamTableDefinitionBuilder.java

```diff
@@ -158,7 +158,9 @@ public TableDefinition build() {
                     appendHeavyAndReadLight();
                 }
                 if (dbSideCompressionForBlocks) {
-                    explicitCompressionBlockSizeKB(Integer.highestOneBit(GenericStreamStore.BLOCK_SIZE_IN_BYTES / 2));
+                    int streamStoreValueSizeKB = GenericStreamStore.BLOCK_SIZE_IN_BYTES / 1_000;
```

Comment:
curious why not using kibibytes here and dividing by 1024?

Reply:
the block size is a constant that is not safe to change

Reply:
yeah, given that this aggressively doesn't work we should probably just delete this.